example/build.gradle

```diff
@@ -0,0 +1,46 @@
+apply plugin: 'com.android.library'
```

Comment:
@JakeWharton When I change the `example` module into a java library, I could not use `apt` plugin anymore. I create a `provided` configuration, add it into the compile classpath, and replace the `apt` with `provided`, just as other guys doing:
```gradle
apply plugin: 'java'

configurations {
    provided
}

sourceSets {
    main {
        compileClasspath += configurations.provided
    }
}

dependencies {
    compile 'com.google.auto.value:auto-value:1.2'
    compile project(':auto-value-gson-annotations')
    provided project(':auto-value-gson-processor')

    testCompile 'junit:junit:4.12'
    testCompile 'org.skyscreamer:jsonassert:1.3.0'
}
```

It works well when I execute command line task `./gradlew build`, I could also find the generated .java file in `example/build/classes/main/`, but Android Studio couldn't find the reference of generated class, do I miss something?